system/HTTP/URI.php

```diff
@@ -154,16 +156,9 @@ class URI
 	 */
 	public function __construct(string $uri = null)
 	{
-		if ( ! is_null($uri))
+		if (!is_null($uri))
```

Comment:
Is it a setting that's automatically removing a space after the exclamation mark? I find it keeps them from getting lost while scanning text to keep the space in, and that matches how the rest of the application has been put together so far.  I don't believe the style guide touches on this particular aspect.